<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MUMTEWD0500" name="Command line test 08f6d5de-f126-4aa5-ab0f-ae3135d71f9c" tests="3" failures="1" timestamp="23 Mar 2018 07:12:32 GMT" time="22.66" errors="0">
  <testcase name="openBrowser" time="8.663" classname="com.HReckAutomation.HReck.HReckLogin"/>
  <testcase name="hReckLogin" time="13.92" classname="com.HReckAutomation.HReck.HReckLogin">
    <failure type="java.lang.AssertionError" message="Test case passed expected [Enter valid password] but found [http://hreck.techeasesystems.in/#/login]">
      <![CDATA[java.lang.AssertionError: Test case passed expected [Enter valid password] but found [http://hreck.techeasesystems.in/#/login]
	at com.HReckAutomation.HReck.HReckLogin.hReckLogin(HReckLogin.java:75)
	at com.mentorstudies.automationframework.TestRunner.main(TestRunner.java:117)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:294)
	at java.lang.Thread.run(Thread.java:748)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- hReckLogin -->
  <testcase name="hReckLogin" time="0.063" classname="com.HReckAutomation.HReck.HReckLogin"/>
</testsuite> <!-- Command line test 08f6d5de-f126-4aa5-ab0f-ae3135d71f9c -->
